### PR TITLE
[new release] grid (0.1.0)

### PR DESCRIPTION
**Pull request by backtracking**
A tiny library for two-dimensional arrays

- Project page: <a href="https://github.com/backtracking/grid">https://github.com/backtracking/grid</a>
- Documentation: <a href="https://backtracking.github.io/grid">https://backtracking.github.io/grid</a>

##### CHANGES:

  - first release

### CODE REVIEW

File: packages/grid/grid.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Jean-Christophe.Filliatre@cnrs.fr"
+authors: "Jean-Christophe Filliâtre"
+synopsis: "A tiny library for two-dimensional arrays"
+description: "Elementary functions over two-dimensional arrays. Not a linear algebra library."
+license: "LGPL-2.1"
```
Comment (haochenx) [inline packages/grid/grid.0.1.0/opam:6]: Do you want `LGPL-2.1-only` or `LGPL-2.1-or-later` ?
ref: https://spdx.org/licenses/

File: packages/grid/grid.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+homepage: "https://github.com/backtracking/grid"
+doc: "https://backtracking.github.io/grid"
+bug-reports: "https://github.com/backtracking/grid/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "2.0.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (haochenx) [inline packages/grid/grid.0.1.0/opam:15]:
```suggestion
  ["dune" "subst"] {dev}
```

This is likely due to [a bug that is fixed in Dune 2.7](https://github.com/ocaml/dune/blob/main/CHANGES.md#270-13082020)

**Review by haochenx — COMMENTED** [review]
There're a couple of opam file linter errors and besides of those this PR looks perfect to me.

File: packages/grid/grid.0.1.0/opam (hunk 1 above)
Comment (backtracking) [reply packages/grid/grid.0.1.0/opam]: fixed (to LGPL-2.1-or-later)

File: packages/grid/grid.0.1.0/opam (hunk 2 above)
Comment (backtracking) [reply packages/grid/grid.0.1.0/opam]: fixed

**Comment by backtracking** [comment]
Thanks @haochenx for the review and the feedback!

**Review by haochenx — APPROVED** [review]
Thanks! This PR now looks perfect to m.

**Comment by mseri** [comment]
Thanks

**Pull request merged**